app/_locales/en/messages.json

```
@@ -2734,6 +2734,9 @@
   "mismatchedRpcUrl": {
     "message": "According to our records the submitted RPC URL value does not match a known provider for this chain ID."
   },
+  "browserDoesntSupportHWandSnaps": {
```

Hmm, this is a strange choice in name (not exactly camel case). Maybe this would be better:

```suggestion
  "noHardwareWalletsOrSnapsSupport": {
```

---

changed in 19326715bb